scss/_utilities_layout.scss

```diff
@@ -4,4 +4,14 @@
   .u-fixed-width {
     @extend %fixed-width-container;
   }
+
+  @if ($table-layout != 'fixed') {
+    .u-table-layout--fixed {
+      table-layout: fixed;
```

Comment:
Utiliities should always have `!important` this allows then to override any pattern styling

Reply:
fixed in #2387